docs/DeveloperGuide.md

```diff
@@ -1,37 +1,1036 @@
+2. Grow My Muscles
+----------------------------------------------------------------------
+```
+**(Steps 10 to 11)** The `PlanList#listAllPlan()` method returns to the `PlanCommand` object
+and the `PlanCommand` object returns to the `WerkIt` object.
+<br><br>
+This completes the process of displaying all plans in WerkIt!.
+
+---
+### Schedule
+User are able to create a 7-day workout plan schedule using the WerkIt application. For each day, user are only allowed
+to schedule 1 workout plan. 
+
+When WerkIt is running, the `WerkIt` class will keep prompting the user to enter command through the
+`WerkIt#startContinuousUserPrompt()` method. After the user has entered command, The `UI#getUserInput()` method in `UI`
+class will catch the user input, and it will be sent to `Parser#parseUserInput(String userInput)` method to analyse the
+user's command. 
+
+If the user's command type is schedule, the `Parser#parseUserInput(String userInput)` method will parse the 'schedule' 
+base word and proceed to create schedule related command using `Parser#createScheduleCommand(String userInput)` method.
+The following table shows the schedule commands that WerkIt! are able to process
+
+| Command                                                             | `<userAction>` | Parameters Meaning                                                                                                                                   |
+|---------------------------------------------------------------------|----------------|------------------------------------------------------------------------------------------------------------------------------------------------------|
+| [schedule /update `<day number>` `<plan number>`](#update-schedule) | update         | `<day number>` Number representing the day. <br/>`<plan number>` Index of the plan stored in planList. This is the plan to be scheduled for the day. |
+| [schedule /list](#view-schedule)                                    | list           |                                                                                                                                                      |
+| [schedule /clear `<day number>`](#clear-schedule-for-a-day)         | clear          | `<day number>` Number representing the day.                                                                                                          |
+| [schedule /clearall](#clear-schedule-for-the-week)                  | clearall       |                                                                                                                                                      |
+
+The `<day number>` range from 1 to 7. The meaning of each day number is explained in the table below.
+
+| Day Number | Meaning   |
+|------------|-----------|
+| 1          | Monday    |
+| 2          | Tuesday   |
+| 3          | Wednesday |
+| 4          | Thursday  |
+| 5          | Friday    |
+| 6          | Saturday  |
+| 7          | Sunday    |
```

Comment:
This seems to be rather trivial to list, perhaps just a line saying that 1 corresponds to Monday will do?